docs/user_guide/method/method.rst

```diff
@@ -0,0 +1,2384 @@
+illustrated in the sample code below.
+
+.. plot::
+    :context: close-figs
+
+
+    import numpy as np
+    from mrsimulator.method.query import RotationalQuery
+    rot_query = RotationalQuery(angle=np.pi/2, phase=0)
+    rot_mixing = MixingEvent(query={"ch1": rot_query})
+
+
+Frequency Contributions
+-----------------------
+
+The NMR frequency, :math:`\Omega(\Theta,i,j)`, of an :math:`i  \rightarrow  j`
+transition between the eigenstates of the stationary-state semi-classical
+Hamiltonian in a sample with a lattice spatial orientation, :math:`\Theta`, can
+be written as a sum of components,
+
+.. math::
+    \Omega(\Theta,i,j) = \sum_k \Omega_k(\Theta,i,j)
+
+with each component, :math:`\Omega_k(\Theta,i,j)`, separated into three parts:
+
+.. math::
+    \Omega_k(\Theta,i,j) = \omega_k \, {\Xi}^{(k)}_L (\Theta) \,{\xi}^{(k)}_\ell (i,j),
+
+where :math:`{\xi}^{(k)}_\ell(i,j)` are the spin transition symmetry functions
+described earlier, :math:`{\Xi}^{(k)}_L(\Theta)` are the spatial symmetry
+functions, and :math:`\omega_k` gives the size of the kth frequency component.
+The experimentalist indirectly influences a frequency component :math:`\Omega_k`
+by direct manipulation of the quantum transition, :math:`i \rightarrow  j`, and
+the spatial orientation,  :math:`\Theta` of the sample.
+
+The function symbol :math:`\Xi_\ell(\Theta)` is similarly replaced with the
+upper-case symbols :math:`\mathbb{S}`, :math:`\mathbb{P}(\Theta)`,
+:math:`\mathbb{D}(\Theta)`, :math:`\mathbb{F}(\Theta)`,
+:math:`\mathbb{G}(\Theta)`, :math:`\ldots`, i.e., following the spectroscopic
+sub-shell letter designations for :math:`L`. Consult the `Symmetry Pathways
+paper <https://doi.org/10.1016/j.pnmrs.2010.11.003>`_ for more details on the
+form of the spatial symmetry functions.  In short, the :math:`\mathbb{S}`
+function is independent of sample orientation, i.e., it will appear in all
+isotropic frequency contributions.  The :math:`\mathbb{D}(\Theta)` function has
+a second-rank dependence on sample orientation, and can be averaged away with
+fast magic-angle spinning, i.e., spinning about an angle, :math:`\theta_R`, that
+is the root of the second-rank Legendre polynomial ":math:`P_2(\cos \theta_R)`.
+The other spatial symmetry functions are removed by spinning the sample about
+the corresponding root of the Lth-rank Legendre polynomial ":math:`P_L(\cos
+\theta_R)`.
+
+For 2nd-order quadrupolar coupling contributions, it is convenient to define
+"hybrid" spin transition functions as linear combinations of the spin transition
+functions
+
+.. math::
+
+    \mathbb{c}_0  = \,\,\,\frac{4}{\sqrt{125}} \, [I(I+1) - 3/4] \, \mathbb{p}_I  + \sqrt{\frac{18}{25}} \, \mathbb{f}_I
+
+.. math::
+
+    \mathbb{c}_2  = \,\,\,\frac{2}{\sqrt{175}} \, [I(I+1) - 3/4] \, \mathbb{p}_I  - \frac{6}{\sqrt{35}} \, \mathbb{f}_I
+
+.. math::
+
+    \mathbb{c}_4  = -\frac{184}{\sqrt{875}} \, [I(I+1) - 3/4] \, \mathbb{p}_I  - \frac{17}{\sqrt{175}} \, \mathbb{f}_I
```

Comment:
This seems a bit out of place or perhaps rushed. I guess there is no easy way to combine the theory and mrsimulator object use.

Reply:
the c_n are in the code, but not used in Method.  It may seem out of place, but I wanted to use the idea of symmetry echoes with frequency averaging.   A little later, I mention that MQ-MAS is a c4 echo.

Reply:
Yeah, I read that. It still felt that a lot is dumped into a small subsection. It is clear right up before this section. I can follow it, but it might not be the same for a new user.

Reply:
For SymmetryQuery objects I didn't see any need (at the moment) to introduce more symmetry functions than p_I and d_I.

But when I started writing about Frequency Contributions, Multiple Events, and Average Frequencies, it became apparent that there needed to be some discussion and motivation.   I added a table with the expressions for each contribution in terms of the symmetry functions, and the only functions not explained were c_0, c_2, and c_4.